qabelbox/src/main/java/de/qabel/qabelbox/activities/BaseActivity.kt

```diff
@@ -0,0 +1,32 @@
+package de.qabel.qabelbox.activities
+
+import android.os.Bundle
+import android.os.PersistableBundle
+import android.support.v7.app.AppCompatActivity
+import de.qabel.qabelbox.QabelBoxApplication
+import de.qabel.qabelbox.dagger.components.ApplicationComponent
+import de.qabel.qabelbox.listeners.IntentListener
+import de.qabel.qabelbox.listeners.toIntentFilter
+
+abstract class BaseActivity : AppCompatActivity() {
+
+    protected open val intentListeners = emptyList <IntentListener>()
```

Comment:
I assume the space after `emptyList` violates our code style ... if we have one in kotlin. At least I would remove it if I saw it so we should agree on one or the other ^^